.stylelintrc.json

```diff
@@ -3,6 +3,7 @@
   "rules": {
     "selector-attribute-quotes": null,
     "no-descending-specificity": null,
-    "declaration-no-important": null
+    "declaration-no-important": null,
+    "selector-type-no-unknown": null
```

Comment:
Do you think there is a way to make it understand that 'figure-inline' exist instead of globally disabling this rule?